site/app/templates/notebook/Notebook.twig

```diff
@@ -1,4 +1,5 @@
 <div class="notebook" id="notebook-main-view">
+    <span class="grading_label"> Notebook View </span>
```

Comment:
This doesn't look great on the student end; it shouldn't be here unless the notebook is being graded. 
![image](https://github.com/Submitty/Submitty/assets/34077605/a95ac42b-1ecf-4c19-9790-ef96442e52fd)

Reply:
ok i will update it